mungegithub/mungers/submit-queue.go

```diff
@@ -499,6 +499,7 @@ func (sq *SubmitQueue) internalInitialize(config *github.Config, features *featu
 		if sq.BatchEnabled {
 			http.Handle("/batch", gziphandler.GzipHandler(http.HandlerFunc(sq.serveBatch)))
 		}
+		// this endpoint is useless without access to prow
 		if sq.ProwURL != "" {
 			http.Handle("/ci-status", gziphandler.GzipHandler(http.HandlerFunc(sq.serveCIStatus)))
```

Comment:
script.js should function. I agree that it might make more sense to serve this path.
Should we also serve /batch regardless?